web/buysell.jsp

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<link rel="stylesheet" type="text/css" href="deptstyle.css">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+    <ul>
+        <li><a class="active" href="index.html">Home</a></li>
+        <li><a href="depts.html">Department/Classes</a></li>
+        <li><a href="login.html">Login/Sign Up</a></li>
+        <li><a href="buysell.html">Buy/Sell</a></li>
+        <li><a href="contact.html">Contact</a></li>
+    </ul>
+    <h1>Buy/Sell Front Page</h1>
+    <table class="fixed">
+        <tr>
+            <th>Post Title</th>
+            <th>Author</th>
+            <th>Class</th>
+            <th>Date</th>
+        </tr>
+        <%@page import = "java.sql.*" %>
+        <%                 
+                Class.forName("com.mysql.jdbc.Driver");
+                Connection con = DriverManager.getConnection("jdbc:mysql://localhost:3306/test_db", "root", "BlackCat13");
+                Statement st = con.createStatement();
+                ResultSet rs;
+                rs = st.executeQuery("SELECT * FROM forum");
+               while (rs.next()) { %>
+                   <tr>
+                        <td><a href="thread_page.jsp?id=<%out.println(rs.getString("threadid"));%>"><%out.println(rs.getString("title"));%></a></td>
```

Comment:
@rsingh01 first this buy/sell page loops through all existing threads in the thread database by using rs.next(), and I create a link for each entry that has the format: "thread_page.jsp?id=#" where # is the threadid that is stored in the database. Every single link will go to the same file, "thread_page.jsp" except the extra information "id" is going to be passed over to the next page.